wordpress-mobile/WordPress-iOS
# Dark mode: remove shade label from semantic color functions

Refs #12320

This is a small QoL change to remove the `shade:` argument label from the semantic color functions. eg. `neutral(shade: .shade0)` becomes `neutral(.shade0)`. I wish the shade names could just be numbers, but since they can't, I now think removing the redundant argument label for the parameter is the right way to go.

To test:
- make sure this builds
- read the code a bit
- look around the app, convince yourself nothing bad happened

Update release notes:
